src/components/program-progress/ProgramProgressCourses.jsx

```diff
@@ -217,21 +345,46 @@ const ProgramProgressCourses = ({ courseData }) => {
       {coursesCompleted?.length > 0
       && (
         <div className="mb-6 courses">
-          <h4 className="white-space-pre"> COURSES COMPLETED    {coursesCompleted.length}</h4>
+          <h4 className="white-space-pre">
+            <FormattedMessage
+              id="enterprise.dashboard.programs.about.page.completed.courses"
+              defaultMessage="COURSES COMPLETED    {completedCourseCount}"
```

Comment:
@muhammad-ammar Yes, the space was intentional as it was present in the original string and I retained it in the marked string.
However, I have moved the space outside the FormattedMessage since it is required.